examples/matplotlib/mpl_plot_lm.py

```diff
@@ -0,0 +1,24 @@
+"""
+Regression Plot
+===============================
+
+_thumb: .6, .5
+_example_title: Plot regression
+"""
+import matplotlib.pyplot as plt
+import xarray as xr
+import numpy as np
+import arviz as az
+
+az.style.use("arviz-darkgrid")
+
+data = az.load_arviz_data("regression1d")
+x = xr.DataArray(np.linspace(0, 1, 100))
+data.add_groups({"constant_data": {"x1": x}})
```

Comment:
Not required just asking. Should we just moditfy regressionid to include these?

Reply:
regression1d needs to be modified a bit. `y` dimensions and `y_model` dimensions do not match. y is aginst [0,100] while latter one is [0,1]